examples/federated-learning-surface-defect-detection-aggregation.Dockerfile

```diff
@@ -7,9 +7,11 @@ COPY ./lib/requirements.txt /home
 
 RUN pip install -r /home/requirements.txt
 
-ENV PYTHONPATH "/home/lib"
+ENV PYTHONPATH "/home/lib:/home/plato"
 
 COPY ./lib /home/lib
+COPY ./plato /home/plato
```

Comment:
better to add plato as the requirement of sedna-lib instead of expose to the user.

Reply:
Due to the large package issue, we currently add Plato as third-party lib.